src/Ardalis.Extensions/Enumerable/IsEmpty.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace Ardalis.Extensions.Enumerable
+{
+    public static partial class EnumerableExtensionMethods
+    {
+        /// <summary>
+        /// Checks if a given enumerable is empty. 
+        /// </summary>
+        /// <typeparam name="T">The generic type of the enumerable.</typeparam>
+        /// <param name="input">True if enumerable is empty otherwise false </param>
+        /// <returns></returns>
+        public static bool IsEmpty<T>(this IEnumerable<T> input) =>
+            (input == null)
+            ? true
+            : false;
```

Comment:
An ienumerable can be empty without being null. I would assume that this extension method would be the equivalent of placing a ! operator on Linq .Any()

Reply:
Agreed. If it's null, return false.

If it's not null, return `!input.Any()`